consumer/consumererror/signalerrors.go

```diff
@@ -35,8 +35,8 @@ func NewTraces(err error, failed ptrace.Traces) error {
 	}
 }
 
-// GetTraces returns failed traces from the associated error.
-func (err Traces) GetTraces() ptrace.Traces {
+// Traces returns failed traces from the associated error.
+func (err Traces) Traces() ptrace.Traces {
```

Comment:
Need to follow the depreciation policy in the contribution readme

Reply:
Whoops, not sure what happened there. Fixed.